docs/source/embedding.md

```diff
@@ -50,6 +50,57 @@ The second option, `Download Widget State`, triggers the downloading of a JSON
 file containing the serialized state of all the widget models currently in use,
 corresponding to the same JSON schema.
 
+## Python interface
+
+Embeddable code for the widgets can also be produced from the Python side.
+The following functions are available in the module `ipywidgets.widgets.embed`:
+
+```rst
+.. embed_snippet::
+
+    s1, s2 = IntSlider(max=200, value=100), IntSlider(value=40)
+    print(embed_snippet(views=[s1, s2]))
```

Comment:
oh, right. I was just cutting and pasting. Well, it wouldn't hurt to have the one import line in the first example.